tests/libvmi/cpu.go

```diff
@@ -69,3 +71,15 @@ func WithNUMAGuestMappingPassthrough() Option {
 		vmi.Spec.Domain.CPU.NUMA = &v1.NUMA{GuestMappingPassthrough: &v1.NUMAGuestMappingPassthrough{}}
 	}
 }
+
+func WithHypervPassthrough() Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		if vmi.Spec.Domain.Features == nil {
+			vmi.Spec.Domain.Features = &v1.Features{}
+		}
+		if vmi.Spec.Domain.Features.HypervPassthrough == nil {
+			vmi.Spec.Domain.Features.HypervPassthrough = &v1.HyperVPassthrough{}
+		}
+		vmi.Spec.Domain.Features.HypervPassthrough.Enabled = virtpointer.P(true)
+	}
+}
```

Comment:
I am not sure this is located in the right file :)

Reply:
Maybe just create a function inside hyperv_test.go file as per `withReEnlightenment`?

Reply:
Yeah, we should introduce these hypervisor features into libvmi, but you're right, cpu is not the ideal pleace for it.
We can handle this in a follow-up :) thanks!

> Maybe just create a function inside hyperv_test.go file as per withReEnlightenment?

Done!